From 44740d089c63446c21314701dc3bb5046629ff6c Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dinuxbg@gmail.com>
Date: Thu, 21 Jul 2016 22:45:01 +0300
Subject: [PATCH 13/17] Disable LOOP instruction generation by default

Keep it disabled by default, but still allow
explicit enabling via -mloop option.

Signed-off-by: Dimitar Dimitrov <dinuxbg@gmail.com>
---
 gcc/common/config/pru/pru-common.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gcc/common/config/pru/pru-common.c b/gcc/common/config/pru/pru-common.c
index 11a83ba46d3..6ee9056ea19 100644
--- a/gcc/common/config/pru/pru-common.c
+++ b/gcc/common/config/pru/pru-common.c
@@ -27,8 +27,11 @@ along with GCC; see the file COPYING3.  If not see
 #include "opts.h"
 #include "flags.h"
 
+#if 0
+/* Disable LOOP usage by default, until it is stable */
 #undef TARGET_DEFAULT_TARGET_FLAGS
 #define TARGET_DEFAULT_TARGET_FLAGS		(MASK_OPT_LOOP)
+#endif
 
 /* Implement TARGET_OPTION_OPTIMIZATION_TABLE.  */
 static const struct default_options pru_option_optimization_table[] =
-- 
2.11.0

